static/schemas/2.0/skus.2.0.json

```diff
@@ -0,0 +1,248 @@
+                    "ipv4": {
+                        "type": "string",
+                        "format": "ipv4"
+                    },
+                    "ipv6": {
+                        "type": "string"
+                    },
+                    "is": {
+                        "type": "string"
+                    },
+                    "lowercase": {
+                        "type": "string"
+                    },
+                    "max": {
+                        "type": "string"
+                    },
+                    "maxLength": {
+                        "type": "string"
+                    },
+                    "min": {
+                        "type": "string"
+                    },
+                    "minLength": {
+                        "type": "string"
+                    },
+                    "notRegex": {
+                        "type": "string"
+                    },
+                    "notContains": {
+                        "type": "string"
+                    },
+                    "notIn": {
+                        "type": "string"
+                    },
+                    "notNull": {
+                        "type": "string"
+                    },
+                    "numeric": {
+                        "type": "string"
+                    },
+                    "required": {
+                        "type": "string",
+                        "format": "date"
+                    },
+                    "regex": {
+                        "type": "string"
+                    },
+                    "truthy": {
+                        "type": "string"
+                    },
+                    "uppercase": {
+                        "type": "string"
+                    },
+                    "url": {
+                        "type": "string",
+                        "format": "url"
+                    },
+                    "urlish": {
+                        "type": "string"
+                    },
+                    "uuid": {
+                        "type": "string"
+                    },
+                    "uuidv3": {
+                        "type": "string"
+                    },
+                    "uuidv4": {
+                        "type": "string"
+                    }
+                }
+            }
+        },
+        "SkusUpsert": {
+            "description": "A sku for RackHD",
+            "type": "object",
+            "required": [
+                "name"
+            ],
+            "properties": {
+                "name": {
+                    "type": "string"
+                },
+                "rules": {
+                    "$ref": "#/definitions/SkusRules"
+                },
+                "discoveryGraphName": {
+                    "type": "string"
+                },
+                "discoveryGraphOptions": {
+                    "type": "object",
+                    "properties": {
+                        "username": {
```

Comment:
They aren't required properties so that shouldn't really matter because users can add or not use those fields. 
I just figured they are commonly used so I'd specify if they're added they would need to be a string.